client/app/templates/account.jade

```diff
@@ -102,6 +102,13 @@
                         .account-form-error-text= t('account 2fa error')
 
             #2fa-infos.account-field
+                p#2fa-strategy
+                
+                p.bold= t('account 2fa rectokens')
```

Comment:
Hey @babolivier ! one day, I'll teach you `<strong>` and `<em>` tags :stuck_out_tongue:.

Btw, not a problem here: we already know the frontend markup isn't clean enough, we'll fix that later :smile:.